changelog.d/5686.feature

```diff
@@ -0,0 +1 @@
+Correction of #5686. Use `M_USER_DEACTIVATED` instead of `M_UNKNOWN` for errcode when a deactivated user attempts to login.
```

Comment:
> Correction of #5686

this is 5686?

If we're updating an existing PR whose changes haven't yet been released, just copy the change fragment so that the entries are merged.

Reply:
It's been released in an rc now :)